### DISCUSSION TITLE
How to change default shell

### DISCUSSION THREAD

**Discussion by nico-bachner** [GitSquared/edex-ui#958] [Questions] [answered]
Is it possible to change the default shell for edex-ui? For example, I would like to set it to open with the fish shell (/usr/local/bin/fish)

**Comment by GitSquared** [comment, marked as answer]
Yep! Check the settings (Ctrl+shift+S). More detailed info in the [wiki](https://github.com/GitSquared/edex-ui/wiki/settings.json).

**Reply by nico-bachner** [reply]
Thank you
